src/LifeCycle.hs

```diff
@@ -70,8 +69,6 @@ thawPhase now = (phaseStatus     %~ thawStatus)
 phaseTrans :: Phase -> PhaseChange -> Maybe (Phase, [PhaseAction])
 phaseTrans (PhaseRefinement ActivePhase{}) RefinementPhaseTimeOut
     = Just (PhaseJury, [JuryPhasePrincipalEmail])
-phaseTrans (PhaseRefinement ActivePhase{}) RefinementPhaseMarkedByModerator
```

Comment:
Removing this does not do any harm. But it is defined in the requirements:
https://media.taiga.io/attachments/3/c/3/3/f2ddcaf7f28d89e2cfccdc793c519951a79b17c9797e84ee01b6ad195bab/160212_aula_process_overview_user_flows.pdf

    NTH: if a global switch is set, instead of
    a fixed duration for the phase, the
    moderator will mark a topic as done so
    it can move to the jury phase"